setup.py

```diff
@@ -11,6 +11,13 @@
             " install setuptools).")
     sys.exit(1)
 
+with open('requirements.txt') as requirements_file:
```

Comment:
Ah yes, good catch. It will need to be added to the python 2.4 compile skip list:

https://github.com/ansible/ansible/blob/devel/test/compile/python2.4-skip.txt